Model/Api/ShippingMethods.php

```diff
@@ -468,7 +468,9 @@ function ($el) {
 
             $cacheIdentifier .= '_' . $quote->getStoreId();
 
+            // phpcs:disable
```

Comment:
should we remove these comments or are they necessary?

Reply:
They are necessary - they are what I referred to in my above comment:
> I added annotated exceptions for our current md5 usages.

Codesniffer ignores the style between `disable` and `enable`.